<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>931</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DotR Randomiser Alpha 0.2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="loadBtn">
        <property name="text">
         <string>Load iso</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Seed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="seedBox"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="genSeedBtn">
        <property name="text">
         <string>Generate seed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="randomiseBtn">
        <property name="text">
         <string>Randomise iso</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QCheckBox" name="aiCheckbox">
        <property name="text">
         <string>Give everyone DMK's AI</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="dcCheckbox">
        <property name="text">
         <string>Disable Deck Cost check</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QCheckBox" name="starterCheckbox">
        <property name="text">
         <string>Randomise starter decks</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>931</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
